data/data.json

```diff
@@ -1350,6 +1350,106 @@
+                  ],
+                  "7119762":[
+                     "te'amim yerukim",
+                     "Organic food"
+                  ],
+                  "7119595":[
+                     "Ra'ash mekomi",
+                     "Local Noise"
+                  ],
+                  "7120870":[
+                     "Festival Israel",
+                     "Towards Israel Festival 2016"
+                  ],
+                  "7120752":[
+                     "80 Shana le'kol Israel",
+                     "80 years to the Israel Broadcasting authorities"
+                  ],
+                  "7060558":[
+                     "atzmaut 68",
+                     "68th Independence Day"
+                  ],
+                  "7120725":[
+                     "shirim le'zichram 2016",
+                     "Evening of songs to commemorate the fallen"
+                  ],
+                  "7093194":[
+                     "Te'amim - hamitbach habari",
+                     "The healthy kitchen"
+                  ],
+                  "7120717":[
+                     "Shirei ha'eurovision",
+                     "Songs from the Eurovision 2016"
+                  ],
+                  "7119018":[
+                     "Or shel kochavime",
+                     "Star Light - series on Hebrew songs - producers and singers"
+                  ],
+                  "7120057":[
+                     "Sikum shana bakolnoa haisraeli",
+                     "Year end summary for the Israeli movie industry"
+                  ],
+                  "7119944":[
+                     "Achshav hator lahava",
+                     "Concert of the songs of Uzi Chitman"
+                  ],
+                  "7119714":[
+                     "festioval mesaprei hasipurim",
+                     "Festival of the story tellers 2014"
+                  ],
+                  "7064392":[
+                     "Shum, pilpel veshemen",
+                     "Series on the sources of the Jewish food - Venice"
+                  ],
+                  "7067328":[
+                     "Shum, pilpel veshemen",
+                     "Series on the sources of the Jewish food - Greece"
+                  ],
+                  "7064471":[
+                     "ha'tsolelet hatsuba",
+                     "The Yellow Submarine - series on the music industry in Jerusalem"
+                  ],
+                  "84":[
+                     "Sod hadvarim hapshutim",
+                     "Amnon Dankner joins with program guests"
+                  ],
+                  "7121337":[
+                     "Gesher tzar me'od",
+                     "The glass bridge designed by Haim Dotan"
+                  ],
+                  "7121375":[
+                     "Rio",
+                     "Rio - the war for peace"
+                  ],
+                  "7064806":[
+                     "Teda kol ek",
```

Comment:
You are right it should be em (mother)

On Tue, Aug 9, 2016 at 11:20 PM, quartern <notifications@github.com> wrote:

> In data/data.json
> <https://github.com/quartern/iltv/pull/47#discussion_r74183491>:
>
> > +                     "The Yellow Submarine - series on the music industry in Jerusalem"
> > +                  ],
> > +                  "84":[
> > +                     "Sod hadvarim hapshutim",
> > +                     "Amnon Dankner joins with program guests"
> > +                  ],
> > +                  "7121337":[
> > +                     "Gesher tzar me'od",
> > +                     "The glass bridge designed by Haim Dotan"
> > +                  ],
> > +                  "7121375":[
> > +                     "Rio",
> > +                     "Rio - the war for peace"
> > +                  ],
> > +                  "7064806":[
> > +                     "Teda kol ek",
>
> Should "ek" be "em"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/quartern/iltv/pull/47/files/e8141f71ff7af13dfa297ffc9401b4460e402bf4#r74183491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASozPhBMQHfamVP5zKyMD6Rkwd_9z9c6ks5qeVGHgaJpZM4JesJd>
> .
>